core/client-internal/src/test/java/tachyon/client/file/BaseFileSystemTest.java

```diff
@@ -69,7 +69,6 @@
    */
   @Before
   public void before() {
-    ClientContext.reset();
```

Comment:
sorry I mean here
```java
Whitebox.setInternalState(mFileSystem, "mContext", mFileContext);
```
therefore `FileSystemContext` is not the default one in clientcontext